README.md

```diff
@@ -18,6 +18,34 @@ To get updates, watch the repo and follow the [Twitter account](https://twitter
 
 # Tips list
 
+## {{tip no}} - Passing arguments to callback functions
+> {{date}} by [@minhazav](https://twitter.com/minhazav)
+
+By default you cannot pass arguments to a callback function. For eaxmple:
+```js
+function callback() {
+	console.log('Hi human');
+}
+
+$('#someelem').on('click', callback);
```

Comment:
It may be good to avoid using jQuery in these tips, especially in cases where beginners might not know that jQuery is being used, and assume this syntax is standard javascript